dask/delayed.py

```diff
@@ -510,17 +518,6 @@ def _rebuild(self, dsk, name=None):
         key = replace_name_in_key(self.key, name) if name else self.key
         return Delayed(key, dsk, self._length)
 
-    def __getstate__(self):
-        return tuple(getattr(self, i) for i in self.__slots__)
-
-    def __setstate__(self, state):
-        for k, v in zip(self.__slots__, state):
-            setattr(self, k, v)
```

Comment:
Why drop these methods? Are they equivalent to the default implementation?

Reply:
Yes. I suspect they were a leftover from Python 2, which needed this sort of massaging to pickle classes with ``__slots__``.